src/network/networkd-link.c

```diff
@@ -1972,8 +1982,14 @@ static int link_joined(Link *link) {
                 r = link_set_bridge(link);
                 if (r < 0)
                         log_link_error_errno(link, r, "Could not set bridge message: %m");
+
         }
 
+        /* TODO: check for bridge interface? */
```

Comment:
no, this would just match for the bridge slaves, but you can apply VLAN IDs on the bridge as well. Is there a chance to get the kind of the netdev? Haven't seen a way to get to the NetDev struct so far, but I guess there is a way.